custom_components/kuna/camera.py

```diff
@@ -118,7 +118,7 @@ async def async_added_to_hass(self):
     def _ready_for_snapshot(self, now):
         return self._next_snapshot_at is None or now > self._next_snapshot_at
 
-    async def async_camera_image(self):
+    async def async_camera_image(self, width: int | None = None, height: int | None = None):
```

Comment:
This signature didn't work, this one does instead: https://github.com/kevin-david/kuna/commit/77163ecd057644ce647907251a2fc7937c94cad9